experimental/SparseMPolyRing/SparseMPolyRing.jl

```diff
@@ -24,11 +24,11 @@ import AbstractAlgebra: base_ring, change_base_ring, change_coefficient_ring,
 
 import AbstractAlgebra.Generic: ordering
 
-export MPolyRingSparse, MPolySparse, PolynomialRingSparse
+export SparseMPolyRing, SparseMPoly, sparse_polynomial_ring
```

Comment:
If you want to follow the new naming scheme, it should be at least this:
```suggestion
export SparseMPolyRing, SparseMPolyRingElem, sparse_polynomial_ring
```

Reply:
@thofma @fieker would be good to reach an agreement what to do with this PR resp. this type.... any alternative proposals, or shall we stick with the suggested new names above? I am somewhat wary of `sparse_polynomial_ring`, I think it may confuse people / lead them to incorrect conclusions about MPoly(Ring)

Reply:
Can we discuss it before/after the next meeting?

Reply:
sure

Reply:
Was there already a conversation about this?

Reply:
nope, at least not with me